scripts/autonomous_cross_deduplicator.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Does cross-deduplication for a series of batches. It selects which batch to do
+next and cross-deduplicates it using the functions in lsh.py
+"""
+
+from argparse import ArgumentParser
+import logging
+from pathlib import Path
+import re
+
+from lsh import cumulative_directory_deduplication
+
+def parse_arguments():
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument('--log-level', '-L', type=str, default='info',

```

Comment:
Please put this at the end of the list (after `-P`), after the task-related arguments.